search-app/src/cmr/search/api/generic_association.clj

```diff
@@ -1,40 +1,50 @@
 (ns cmr.search.api.generic-association
   "Defines common functions used by associations among generic concepts in the CMR."
   (:require
-   [cheshire.core :as json]
-   [cmr.common-app.api.enabled :as common-enabled]
-   [cmr.common.concepts :as common-concepts]
-   [cmr.common.log :refer (info)]
-   [cmr.common.mime-types :as mt]
-   [cmr.common.util :as util]
-   [cmr.search.services.generic-association-service :as generic-assoc-service]
-   [compojure.core :refer :all]))
+    [cheshire.core :as json]
+    [cmr.common-app.api.enabled :as common-enabled]
+    [cmr.common.concepts :as common-concepts]
+    [cmr.common.log :refer (info)]
+    [cmr.common.mime-types :as mt]
+    [cmr.common.util :as util]
+    [cmr.search.api.association :as assoc]
+    [cmr.search.services.generic-association-service :as generic-assoc-service]
+    [compojure.core :refer :all]))
 
 (defn- validate-association-content-type
   "Validates that content type sent with a association is JSON."
   [headers]
   (mt/extract-header-mime-type #{mt/json} headers "content-type" true))
 
-(defn- api-response
+(defn api-response
```

Comment:
why is this now public?

Reply:
good catch. I made it public to create a unit test real fast. Need to set it back to private